### ISSUE TITLE
Add support for content:encoded support for Podcast

### ISSUE THREAD

**Issue by rock3r** [mpgirro/stalla#94]
We are modeling the `content:encoded` for the `<item>` nodes, but not for the `<channel>` nodes. That's not great, because many podcasts use it (e.g., `src/test/resources/xml/real-life-feeds/reply-all.xml`). We should add a `PodcastContent`.

**Comment by rock3r** [comment]
I have done this but it is a breaking change both in terms of source and binary compatibility. I can create a 2.0.0 branch once #95 is merged, and we can consider shipping 1.1.0 since it has a meaty set of changes as it is already. What do you think @mpgirro?

**Comment by mpgirro** [comment]
I'm fine with shipping 1.1.0. I've got about half of #76 done, but won't be able to finish it in the near future. Just something for 1.2.0 then :)

**Comment by rock3r** [comment]
Makes sense. I'll tag and release 1.1.0, and create 1.2.0 and 2.0.0 milestones.

**Comment by mpgirro** [comment]
👍

**Comment by rock3r** [comment]
I've discovered a pretty severe bug with the `List<...>.asBuilders()` implementations, I'm working on a fix and will release 1.1 after that's shipped.
